HandyImage.user.js

```diff
@@ -816,6 +816,7 @@
 // @match		http://foxyimg.link/*
 // @match		*://*.imgflash.net/img-*
 // @match		http://*.sexyimagexxx.com/img-*
+// @match   		http://*.imgtornado.com/img-*
```

Comment:
hello? please... do it like the above lines, just copy those lines with "spaces" and write new site address instead of the old one if you don't want to press "TAB" two times for some reason